packages/apollo/src/commands/client/extract.ts

```diff
@@ -9,7 +10,13 @@ import { ClientIdentity } from "apollo-language-server";
 export default class ClientExtract extends ClientCommand {
   static description = "Extract queries from a client";
   static flags = {
-    ...ClientCommand.flags
+    ...ClientCommand.flags,
+    stripLiterals: flags.boolean({
+      description:
+        "Whether to strip literals from extracted queries. DEFAULT: true",
```

Comment:
```suggestion
        "Disable redaction of string and numerical literals.  Without this flag, these values will be replaced with empty strings (`''`) and zeroes (`0`) respectively.  This redaction is intended to avoid inadvertently outputting potentially personally identifiable information (e.g. embedded passwords or API keys) into operation manifests.",
```